src/legacy/core_plugins/kbn_vislib_vis_types/public/editors/point_series.tsx

```diff
@@ -26,9 +26,16 @@ import { SwitchOption } from '../controls/switch';
 import { GridOptions } from '../controls/point_series/grid_options';
 import { BasicOptions } from '../controls/basic_options';
 import { BasicVislibParams } from '../types';
+import { NumberInputOption } from '../controls/number_input';
+import { SelectOption } from '../controls/select';
 
 function PointSeriesOptions(props: VisOptionsProps<BasicVislibParams>) {
   const { stateParams, setValue, vis } = props;
+  const options = [
+    { value: 'full', text: 'Full' },
```

Comment:
@FilipRy The `text` value of each option need to be put through a `i18n.translate` call (as you did below), so they can properly internationalized.